controllers/ibmpowervsmachine_controller.go

```diff
@@ -170,17 +172,12 @@ func (r *IBMPowerVSMachineReconciler) reconcileDelete(scope *scope.PowerVSMachin
 	}()
 
 	if scope.IBMPowerVSMachine.Status.InstanceID == "" {
-		scope.Info("InstanceID is not yet set, hence not invoking the powervs API to delete the instance")
+		scope.Info("InstanceID is not yet set, hence not invoking the PowerVS API to delete the instance")
 		return ctrl.Result{}, nil
 	}
 	if err := scope.DeleteMachine(); err != nil {
 		scope.Info("error deleting IBMPowerVSMachine")
-		return ctrl.Result{}, errors.Wrapf(err, "error deleting IBMPowerVSMachine %s/%s", scope.IBMPowerVSMachine.Namespace, scope.IBMPowerVSMachine.Name)
-	}
-	// Remove the cached VM IP
-	err := scope.DHCPIPCacheStore.Delete(powervs.VMip{Name: scope.IBMPowerVSMachine.Name})
```

Comment:
why we have removed this block? can this be part of separate PR if needed?

Reply:
will add it back in this PR but we can think of removing it in future because once the cache time is over its automatically removed.